packages/playground/_stories/main/StepInput/StepInput.stories.ts

```diff
@@ -88,14 +78,14 @@ ValuePrecision.args = {
 };
 
 export const Label = Template.bind({});
-Label.storyName = "With Label and Alignment";
+Label.storyName = "With Label and Text Alignment";
```

Comment:
We should think for a shorter name as per the task description. In general all input fields should have a label either way. Don't think we should point out that there is an external label to any input component.